src/Components/Landing/Landing.jsx

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable import/no-absolute-path */
+import React from "react";
+// eslint-disable-next-line import/no-unresolved
+import HeroBanner from "./Group.png";
```

Comment:
Instead of importing the image, put the image file in public/assets folder (already created, pull from the `home-page` branch and you will get it

After placing the image in `public/assets/Group.png`

You can use that image as follows 

```html
<img src="/assets/Group.png" alt="Grafiny Landing"/>
```

[References](https://vitejs.dev/guide/assets.html#the-public-directory)

Reply:
done task